frontend/src/components/CreateAccountPage/CreateAccountPage.css

```diff
@@ -0,0 +1,219 @@
+@import url('https://fonts.googleapis.com/css?family=Rubik&display=swap');
+
+#createAccountBox {
+    width: 675px;
+    height: 760px;
```

Comment:
Create account box is taking the full height of the screen for larger screen sizes:
<img width="1440" alt="Screen Shot 2022-01-10 at 10 39 09 PM" src="https://user-images.githubusercontent.com/42549561/148893596-d806efeb-7774-4d8e-8932-3d07b528bdc1.png">
vs. Figma:
<img width="622" alt="Screen Shot 2022-01-10 at 10 39 36 PM" src="https://user-images.githubusercontent.com/42549561/148893658-ac751bb1-97b2-4f37-a134-b045faf3bf35.png">

Reply:
Adjustment done. As for the create account text, would you prefer the bolded version or the non-bolded version?

**Nonbolded:**
![image](https://user-images.githubusercontent.com/71291167/148896775-133de2d6-ae61-433e-90b2-0070eea856dd.png)

**Bolded:**
![image](https://user-images.githubusercontent.com/71291167/148896826-3232dcac-17d6-43b8-9aea-668df2bc3485.png)

Reply:
As a developer, we need aim for consistency whenever possible. By the looks of it (comparing to other screens), probably the bold. What do you think?

Reply:
I came to the same conclusion. Thank you for your suggestions.